sessions/react-effects-and-localstorage/journal-app-localstorage/src/App.js

```diff
@@ -0,0 +1,88 @@
+import { useState } from "react";
+import "./App.css";
+import EntriesSection from "./components/EntriesSection";
+import EntryForm from "./components/EntryForm";
+import Footer from "./components/Footer";
+import Header from "./components/Header";
+import { nanoid } from "nanoid";
+
+const initialEntries = [
+  {
+    id: 1000,
+    date: "Feb 5, 2025",
+    motto: "We are in a state of chaos",
+    notes:
+      "Today I learned about React State. It was fun! I can't wait to learn more.",
+  },
+  {
+    id: 999,
+    date: "Feb 4, 2025",
+    motto: "Props, Props, Props",
+    notes:
+      "Today I learned about React Props. Mad props to everyone who understands this!",
+  },
+  {
+    id: 998,
+    date: "Feb 3, 2025",
+    motto: "How to nest components online fast",
+    notes:
+      "Today I learned about React Components and how to nest them like a pro. Application design is so much fun!",
+  },
+  {
+    id: 997,
+    date: "Feb 2, 2025",
+    motto: "I'm a React Developer",
+    notes: "My React-ion when I learned about React: 😍",
+  },
+];
+
+function App() {
+  const [entries, setEntries] = useState(initialEntries);
+  const [showFavorites, setShowFavorites] = useState(false);
+
+  function handleAddEntry(newEntry) {
+    const date = new Date().toLocaleDateString("en-us", {
+      dateStyle: "medium",
+    });
+    setEntries([{ id: nanoid(), date, ...newEntry }, ...entries]);
+  }
+
+  function handleToggleFavorite(id) {
+    setEntries(
+      entries.map((entry) =>
+        entry.id === id ? { ...entry, isFavorite: !entry.isFavorite } : entry
+      )
+    );
+  }
+
+  function handleShowFavoriteEntries() {
+    setShowFavorites(true);
+  }
+
+  function handleShowAllEntries() {
+    setShowFavorites(false);
+  }
+
+  const favoriteEntries = entries.filter((entry) => entry.isFavorite);
+
+  return (
+    <div className="app">
+      <Header />
+      <main className="app__main">
+        <EntryForm onAddEntry={handleAddEntry} />
+        <EntriesSection
+          entries={showFavorites ? favoriteEntries : entries}
+          showFavorites={showFavorites}
+          amountAllEntries={entries.length}
+          amountFavoriteEntries={favoriteEntries.length}
```

Comment:
I'd call these either `count` or `length` (I am leaning `count`)

```suggestion
          allEntriesLength={entries.length}
          favoriteEntriesLength={favoriteEntries.length}
```

```suggestion
          allEntriesCount={entries.length}
          favoriteEntriesCount={favoriteEntries.length}
```